suite-native/module-connect-device/src/screens/ConnectDeviceCrossroadsScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    AccountsImportStackRoutes,
+    ConnectDeviceStackParamList,
+    ConnectDeviceStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    Screen,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import { VStack, Card, Button, Text, Pictogram } from '@suite-native/atoms';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+
+const cardStyle = prepareNativeStyle<{ flex: 1 | 2 }>((utils, { flex }) => ({
+    flex,
+    justifyContent: 'center',
+    paddingTop: 40,
+    paddingBottom: utils.spacings.extraLarge,
+}));
+
+const textStyle = prepareNativeStyle(_ => ({
+    textAlign: 'center',
+}));
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    ConnectDeviceStackParamList,
+    ConnectDeviceStackRoutes.ConnectDeviceCrossroads,
+    RootStackParamList
+>;
+
+export const ConnectDeviceCrossroadsScreen = () => {
+    const navigation = useNavigation<NavigationProps>();
+    const { applyStyle } = useNativeStyles();
+
+    const handleSyncMyCoins = () => {
+        navigation.navigate(RootStackRoutes.AccountsImport, {
+            screen: AccountsImportStackRoutes.SelectNetwork,
+        });
+    };
+
+    const handleConnectDevice = () => {
+        navigation.navigate(RootStackRoutes.ConnectDevice, {
+            screen: ConnectDeviceStackRoutes.ConnectAndUnlockDevice,
+        });
+    };
+
+    return (
+        <Screen>
+            <VStack spacing="medium" flex={1}>
+                <Card style={applyStyle(cardStyle, { flex: 2 })}>
```

Comment:
IMO the flex style prop is confusing since it does look like an inline style. 

I would prefer to 
```suggestion

                const cardStyle = prepareNativeStyle(_ => ({
                    flex:2
                }));

                ...
                
                <Card style={[applyStyle(cardStyle), applyStyle(doubleFlexStyle)]}>
```

But it is maybe just my personal preference, no big deal if you don't fix.

Reply:
Seems normal to me. It's just normal style prop

Reply:
Ok, no problem.

Reply:
It's just a restriction that's unnecessary. If you want it to just accept number, no problem, I can send fixup